### ISSUE TITLE
"png_get_eXIf_1 ", has anyone encountered this issue? How to deal with it?

### ISSUE THREAD

**Issue by freemank1224** [lattebyte/DepthPro-Windows-GPU#3]
![ree](https://github.com/user-attachments/assets/fd943022-0856-4b6e-9545-f2134b1a14a0)
When I run the code, the above dialog box prompt. After closing the dialog box, it displays:
C:\Users\dyson\miniconda3\envs\depth-pro-win-gpu\lib\site-packages\torchvision\io\image.py:14: UserWarning: Failed to load image Python extension: '[WinError 127] 找不到指定的程序。'If you don't plan on using image functionality from torchvision.io, you can ignore this warning. Otherwise, there might be something wrong with your environment. Did you have libjpegorlibpnginstalled before buildingtorchvision from source? warn(

**Comment by lattebyte** [comment]
Hi freeman, thanks for trying this project, would you mind switch to anaconda and watch if still any errors? https://www.anaconda.com/download/success

**Comment by freemank1224** [comment]
> Hi freeman, thanks for trying this project, would you mind switch to anaconda and watch if still any errors? https://www.anaconda.com/download/success

Thanks! I fixed this issue for me. I deleted everything including virtual env (and its files on my disk), the local files of the project repo. 
Then, I noticed the pytorch components versions were different after using conda install, so I reinstall them using pip install！
Install OpenCV manually, by using `pip install python-contrib-opencv` instead of 'python-opencv'.
And then it works! I don't know where the error exactly lies, but it works now
